dags/infra/variables.py

```diff
@@ -8,6 +8,19 @@
     "index_landsat_scenes_sqs_queue", "deafrica-dev-eks-index-landsat-scene"
 )
 
+AFRICA_AWS_REGION = Variable.get("cape-town-region", "af-south-1")
+
+AFRICA_SNS_TOPIC_ARN = Variable.get(
+    "landsat_sns_topic",
+    "arn:aws:sns:af-south-1:717690029437:deafrica-dev-eks-landsat-topic",
+)
+
+AFRICA_S3_BUCKET_NAME = Variable.get("landsat_bucket_name", "deafrica-landsat-dev")
+
+USGS_S3_BUCKET_NAME = Variable.get("usgs-bucket-name", "usgs-landsat")
```

Comment:
no, those are not from us, we don't have control over it, keep USGS string in ur own folder